src/components/activity/ActivityDirectivesTable.svelte

```diff
@@ -129,11 +137,22 @@
   function getRowId(activityDirective: ActivityDirective): ActivityDirectiveId {
     return activityDirective.id;
   }
+
+  function scrollTimelineToActivityDirective() {
+    const directiveId =
```

Comment:
I don't think you actually have to check `bulkSelectedActivityDirectiveIds` since `selectedActivityDirectiveId` will be set for the primary selection regardless of bulk selection.